Firestore/Source/Public/FIRFirestore.h

```diff
@@ -139,6 +139,24 @@ NS_SWIFT_NAME(Firestore)
 + (void)enableLogging:(BOOL)logging
     DEPRECATED_MSG_ATTRIBUTE("Use FIRSetLoggerLevel(FIRLoggerLevelDebug) to enable logging");
 
+
+#pragma mark - Network
+
+/**
+ * Re-enables usage of the network by this Firestore instance after a prior call to
+ * disableNetworkWithCompletion. Completion block, if provided, will be called once network uasge
+ * has been enabled.
+ */
+- (void)enableNetworkWithCompletion:(void (^ _Nullable)(NSError *_Nullable error))completion;
```

Comment:
Please add a swift language compilation test in main.swift.

Make sure that both `enableNetwork(completion:)` and trailing closure syntax work as expected,.

Reply:
Done.